src/lib/ai/agents/tools/chat/voice-channel/leaveVC.ts

```
@@ -0,0 +1,42 @@
+import { createLogger } from '@/lib/logger';
+import { getVoiceConnection } from '@discordjs/voice';
```

ESLint

[error] Unable to resolve path to module '@discordjs/voice'.

(import-x/no-unresolved)